### ISSUE TITLE
Scenario - Excavation Mammoth doesnt start

### ISSUE THREAD

**Issue by kaellisto** [PrehistoricKingdom/feedback#632]
During the Scenario mode, I am not able to start the excavation for the mammoths. I have restarted the save a few times but now I can not access the Objectives anymore nor start the excavation

**Comment by easywolf32** [comment]
Exactly, same here. Do they even bother answering these questions? Kinda stoopid to release a game in this condition...like 2/10 atm.
